src/ui/public/getting_started/registry.js

```diff
@@ -0,0 +1,78 @@
+import uuid from 'node-uuid';
+
```

Comment:
This looks like a job for `ui/registry`. Nothing in here seems to warrant maintaining another registry.